src/PrestaShopBundle/Entity/Repository/StockRepository.php

```diff
@@ -185,19 +185,65 @@ private function selectStockBy(ProductIdentity $productIdentity)
 
     /**
      * @param QueryParamsCollection $queryParams
+     * @param bool $export
      * @return mixed
      */
-    public function getData(QueryParamsCollection $queryParams)
+    public function getData(QueryParamsCollection $queryParams, $export = false)
     {
         $this->stockManager->updatePhysicalProductQuantity(
             $this->shopId,
             $this->orderStates['error'],
             $this->orderStates['cancellation']
         );
 
-        return parent::getData($queryParams);
+        return parent::getData($queryParams, $export);
     }
 
+    /**
+     * Specific function to export data
+     * @param QueryParamsCollection $queryParams
+     * @return array
+     */
+    public function getDataExport(QueryParamsCollection $queryParams)
+    {
+        $data = $this->getData($queryParams, true);
+
+        $formatedData = array(
+            array(
+                // headers columns
+                'product_id' => 'Product ID',
```

Comment:
'Product ID' and other data from the array: Is this translatable?

Reply:
Yes we can